orchestrator/index.js

```diff
@@ -108,10 +108,7 @@ if (args.kill_idle) {
 }
 //instantiate n executors and producers, passing the executor number as a parameter,
 //to be used as a suffix of the rabbitmq queue name
-let queueNumber = config.amqp.queue_n;
```

Comment:
Since you removed the use of workers, remove it from the list of possible arguments.

Reply:
Ok!